platform/vs/docker-sonic-vs/start.sh

```diff
@@ -59,34 +59,34 @@ rm -f /var/run/rsyslogd.pid
 supervisorctl start rsyslogd
 
 supervisord_cfg="/etc/supervisor/conf.d/supervisord.conf"
-chassis_cfg_file="/usr/share/sonic/virtual_chassis/default_config.json"
-chassis_cfg_file_default="/etc/default/sonic-db/default_chassis_cfg.json"
+globaldb_cfg_file="/usr/share/sonic/virtual_chassis/default_config.json"
+globaldb_cfg_file_default="/etc/default/sonic-db/default_globaldb_cfg.json"
 host_template="/usr/share/sonic/templates/hostname.j2"
 db_cfg_file="/var/run/redis/sonic-db/database_config.json"
 db_cfg_file_tmp="/var/run/redis/sonic-db/database_config.json.tmp"
 
-if [ -r "$chassis_cfg_file" ]; then
-   echo $(sonic-cfggen -j $chassis_cfg_file -t $host_template) >> /etc/hosts
+if [ -r "$globaldb_cfg_file" ]; then
+   echo $(sonic-cfggen -j $globaldb_cfg_file -t $host_template) >> /etc/hosts
 else
-   chassis_cfg_file="$chassis_cfg_file_default"
-   echo "10.8.1.200 redis_chassis.server" >> /etc/hosts
+   globaldb_cfg_file="$globaldb_cfg_file_default"
+   echo "10.8.1.200 redis_global.server" >> /etc/hosts
 fi
 
 mkdir -p /var/run/redis/sonic-db
 cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db/
 
 supervisorctl start redis-server
 
-start_chassis_db=`sonic-cfggen -v DEVICE_METADATA.localhost.start_chassis_db -y $chassis_cfg_file`
-if [[ "$HOSTNAME" == *"supervisor"* ]] || [ "$start_chassis_db" == "1" ]; then
-   supervisorctl start redis-chassis
-   python /usr/bin/chassis_db.py
+start_global_db=`sonic-cfggen -v DEVICE_METADATA.localhost.start_global_db -y $globaldb_cfg_file`
+if [[ "$HOSTNAME" == *"supervisor"* ]] || [ "$start_global_db" == "1" ]; then
+   supervisorctl start redis-global
+   python /usr/bin/global_db.py
 fi
 
-conn_chassis_db=`sonic-cfggen -v DEVICE_METADATA.localhost.connect_to_chassis_db -y $chassis_cfg_file`
-if [ "$start_chassis_db" != "1" ] && [ "$conn_chassis_db" != "1" ]; then
+conn_global_db=`sonic-cfggen -v DEVICE_METADATA.localhost.connect_to_global_db -y $globaldb_cfg_file`
```

Comment:
can we combine the calls to `sonic-cfggen` into one call using a template file? Here is an [example](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry_vars.j2).

Reply:
I'd do something like:
```
SONIC_CFGGEN_PARAMS="
    -v DEVICE_METADATA.localhost.connect_to_global_db \
    -y $globaldb_cfg_file \
    -t globaldb_vars.json.j2,/tmp/globaldb.json
"
if [ -r "$globaldb_cfg_file" ]; then
￼    $SONIC_CFGGEN_PARAMS+=" -j $globaldb_cfg_file -t $host_template,/etc/hosts"
else
.
fi
$(sonic-cfggen $SONIC_CFGGEN_PARAMS)

￼

Reply:
I can the two calls into one call as the example does, one concern is that will require jq tool, is jq available on host and database container?

Reply:
`jq` is available in dockers' nut not on host yet. There was no need to add this support then. please feel free to add it here.